src/domains/ecommerce-app/api/ecommerce-io/v2/_openapi.json.tpl

```diff
@@ -712,6 +712,96 @@
         }
       }
     },
+    "/payment-methods/{id}/redirectUrl": {
+      "get": {
+        "tags": [
+          "ecommerce-payment-methods"
+        ],
+        "operationId": "getMethodRedirectUrl",
+        "summary": "Redirection URL for input payment method",
+        "description": "Return the URL to be followed for a specific method\n",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "schema": {
+              "type": "string"
+            },
+            "required": true,
+            "description": "The ID of the payment method chosen for the payment process"
+          },
+          {
+            "in": "query",
+            "name": "rpt_id",
```

Comment:
Are these query parameters search filters for the payment method subresource? Otherwise, aren't we creating a subresource that requires a POST?